charts/flink-job/README.md

```diff
@@ -51,6 +51,11 @@ See metrics reporter in the flink properties for more details.
 | podMonitor.sampleLimit | int | `2000` | (`int`) Per-scrape limit on number of scraped samples that will be accepted. |
 | podMonitor.scrapeInterval | string | `"15s"` | (`string`) The frequency in which to scrape metrics. |
 | pvc | object | `{"storage":"1Gi","storageClassName":"efs"}` | Configuration of the PersistentVolume for storing savepoints. |
+| pvcRocksdb | object | `{"enabled":false,"rocksdbDir":"/rocksdb","storage":"10Gi","storageClassName":"gp3-encrypted"}` | Configuration of rocksdb PVC |
```

Comment:
Personally I'd have a key named `rocksdb` and inside it I would have the options like this:

```yaml
rocksdb:
  enabled: true|false  # (this would somehow configure the flink job itself to use rocksdb?)
  pvc:
    enabled: true|false
    storageClassName: null  # if set, it would then set it in the template
    storage: 10Gi
    path: /rocksdb
```